src/DicomMetaDictionary.js

```diff
@@ -402,6 +419,68 @@ DicomMetaDictionary.sopClassNamesByUID = {
     "1.2.840.10008.5.1.4.1.1.77.1.5.4": "OphthalmicTomographyImage"
 };
 
+DicomMetaDictionary.encodingMapping = {
```

Comment:
Can you drop this externally as it's own file?  There should also be an addEncoding method to externally add them.
just import the encoding mapping and assign to the dicom meta dictionary rather than increasing the overall contents of this file.